test-cases/gemini/gemini-1tb-10h.yaml

```diff
@@ -23,5 +23,5 @@ gemini_schema_url: 'https://s3.amazonaws.com/scylla-gemini/Binaries/schema.json'
 
 db_type: mixed_scylla
 instance_type_db_oracle: 'i3.16xlarge'
-oracle_scylla_version: '4.3.3'
+oracle_scylla_version: '4.4.7'
```

Comment:
Let’s set it in the test_defaults once and remove it from all these yamls.

Reply:
done